src/Models/IoTWorkbenchProjectBase.ts

```diff
@@ -291,9 +305,25 @@ export abstract class IoTWorkbenchProjectBase {
     try {
       const iotworkbenchprojectFilePath =
           path.join(projectFolder, FileNames.iotworkbenchprojectFileName);
-      if (!await FileUtility.fileExists(type, iotworkbenchprojectFilePath)) {
-        await FileUtility.writeFile(type, iotworkbenchprojectFilePath, '');
+
+      let projectConfig: {[key: string]: string} = {};
+      if (await FileUtility.fileExists(type, iotworkbenchprojectFilePath)) {
+        const projectConfigContent =
+            await FileUtility.readFile(
+                type, iotworkbenchprojectFilePath, 'utf8') as string;
+        if (projectConfigContent !== '') {
+          projectConfig = JSON.parse(projectConfigContent);
+        }
       }
+
+      projectConfig[`${ConfigKey.projectHostType}`] =
+          ProjectHostType[this.projectHostType];
+
+      const indentationSpace = 4;

```

Comment:
For JSON file, the recommended indentation is 2, what's the best practice in our team?

Reply:
Maybe related to gts validation. Considering switch from `gts` to `tslint` for coding style check.